common/src/main/java/org/apache/cassandra/sidecar/common/data/ListSnapshotFilesRequest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.cassandra.sidecar.common.data;
```

Comment:
If the data class is created for `ListSnapshotFilesHandler`, I do not see why placing it under `common` package, since it is not shared by multiple packages. It is coupled with `ListSnapshotFilesHandler`. Maybe relocated it together with the handler?

Reply:
Currently, there is no other project using `ListSnapshotFilesRequest`. However, we do have plans for a Sidecar client project where this classes would be shared between the client and server code